apps/language_server/lib/language_server/providers/completion.ex

```diff
@@ -520,6 +543,7 @@ defmodule ElixirLS.LanguageServer.Providers.Completion do
     |> String.replace("$", "\\$")
     |> String.replace("}", "\\}")
     |> String.split(",")
+    |> Enum.reject(fn s -> String.contains?(s, "\\\\") end)
```

Comment:
I think this would be easier to follow if it was extracted into a small named function, something like:
`defp is_default_argument?(s), do: String.contains?(s, "\\\\")`